fh-mbaas.js

```diff
@@ -223,6 +229,7 @@ function initModules(clusterWorker, jsonConfig, cb) {
     if (!err) {
       return cb();
     }
+    initModulesCalled = false;
```

Comment:
Why set it back to false here? Does that not mean it will be executed every time?